app/assets/javascripts/insights.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Do we really need this empty file in commit?

Reply:
solved